src/ORM/Exception/PersistenceFailedException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Cake\ORM\Exception;
+
+use Cake\Core\Exception\Exception;
+
+/**
+ * Used when a strict save or delete fails
+ */
+class PersistenceFailedException extends Exception
+{
+
+    protected $_messageTemplate = 'Failed to %s %s';
```

Comment:
How will the message look like at the end? Does this show a json representation of the entity?